libs/payments/cart/src/lib/cart.service.ts

```diff
@@ -264,24 +261,16 @@ export class CartService {
     const fxaAccount =
       fxaAccounts && fxaAccounts.length > 0 ? fxaAccounts[0] : undefined;
 
-    let currency: string | undefined = undefined;
-    if (taxAddress?.countryCode) {
-      currency = this.currencyManager.getCurrencyForCountry(
-        taxAddress?.countryCode
-      );
-      if (!currency) {
-        throw new CartInvalidCurrencyError(currency, taxAddress.countryCode);
-      }
-    }
-
-    if (!currency) currency = DEFAULT_CURRENCY;
+    const currency =
+      this.currencyManager.getCurrencyForCountry(args.taxAddress.countryCode) ??
+      DEFAULT_CURRENCY;
```

Comment:
Oh could we remove the `DEFAULT_CURRENCY` before merging this?

If currency can't be determined an error should be thrown.